pkg/os/disk/disk.go

```diff
@@ -127,6 +127,10 @@ func ListDiskLocations() (map[uint32]Location, error) {
 					switch strings.TrimSpace(itemSplit[0]) {
 					case "Adapter":
 						d.Adapter = strings.TrimSpace(itemSplit[1])
+						if d.Adapter == "0" {
```

Comment:
@mweibel FYI. since using cim to list disks would break disk recovery in Win 2019, details: https://github.com/kubernetes-sigs/azuredisk-csi-driver/issues/3029, I would like to use original Get-Disk command to fix this issue (filter out `0` Adapter) to fix this issue. that could also solve the issue: https://github.com/kubernetes-sigs/azuredisk-csi-driver/issues/2661